database/mssql/scripts/dbscripts/PSP_PIMS_S21_00/Alter Down/10_DML_PIMS_STATIC_VARIABLE_VERSION.sql

```diff
@@ -0,0 +1,26 @@
+/* -----------------------------------------------------------------------------
```

Comment:
@FuriousLlama why are all these IS21 dbscripts look like they were introduced by this PR?

Reply:
that's probably git not doing the diff properly on this screen, I will force git to check again